Electron/src/pages/UserProfile/UserProfile.tsx

```diff
@@ -263,31 +270,55 @@ export default function UserProfile({
 
       {userType === UserType.USER && (
         <div className="p-4">
-          <h2
-            style={{
-              color: 'var(--pure-white)',
-              fontWeight: '700',
-              fontSize: '1.5rem',
-              marginTop: '1rem',
-              marginBottom: '1.5rem',
-            }}
-          >
-            Historial de reproducción
-          </h2>
-          <div className="d-flex flex-row flex-wrap " style={{ gap: '14px' }}>
-            {playbackHistory &&
-              playbackHistory.map((songItem, index) => {
-                return (
-                  <SongCard
-                    // eslint-disable-next-line react/no-array-index-key
-                    key={`${songItem.name}-${index}`}
-                    name={songItem.name}
-                    photo={songItem.photo}
-                    artist={songItem.artist}
-                    refreshSidebarData={refreshSidebarData}
-                  />
-                );
-              })}
+          <div className="d-flex">
+            <div className={`w-100 d-flex ${styles.categoryTitleContainer}`}>
+              <button
+                type="button"
+                className={`${styles.categoryTitle}`}
+                style={{
+                  border: 'none',
+                  backgroundColor: 'transparent',
+                  color: 'var(--pure-white)',
+                  fontWeight: '700',
+                  fontSize: '1.5rem',
+                  marginTop: '1rem',
+                  marginBottom: '1.5rem',
+                }}
+                onClick={() => {
+                  handleShowAllUserStreamHistory(userType);
+                }}
+              >
+                Historial de reproducción
+              </button>
+            </div>
+            <div
+              className={`container-fluid d-flex ${styles.mostrarTodoContainer}`}
+            >
+              <button
+                type="button"
+                className={`${styles.mostrarTodo}`}
+                onClick={() => {
+                  handleShowAllUserStreamHistory(userType);
+                }}
+              >
+                Mostrar todos
+              </button>
+            </div>
+            <div className="d-flex flex-row flex-wrap " style={{ gap: '14px' }}>
+              {streamHistory &&
+                streamHistory.slice(0, 5).map((songItem) => {
+                  return (
+                    <SongCard
+                      // eslint-disable-next-line react/no-array-index-key
```

Comment:
Can we delete `// eslint-disable-next-line react/no-array-index-key` and use `songItem.name-songItem.artist` ?